packages/x-charts/src/BarChart/BarChart.tsx

```diff
@@ -25,11 +30,21 @@ export function BarChart(props: BarChartProps) {
 
   return (
     <ChartContainer
-      series={series}
+      series={series.map((s) => ({ ...s, type: 'bar' }))}
```

Comment:
Would it ever make sense to have any different `type` then `bar` here? 🤔

Reply:
No, because this component only use of a `<BarPlot />`

But I could reverse the order

```diff
- series={series.map((s) => ({ ...s, type: 'bar' }))}
+ series={series.map((s) => ({ type: 'bar', ...s }))}
```

Such devs could introduce other types of series. without having the component failing

Reply:
Yeah, that was my main point. Let's maybe not be super strict about the type. 👌